projectforum/projects/views.py

```diff
@@ -44,16 +72,13 @@ class ProjectView(View):
     ascending: Whether or not we sort by ascending or descending order
         True: Ascending order
         *False: Descending order
-    starting_from: Integer Default is 1.  Return projects starting from this
-                   number
-    ending_at: Integer Default is 10. Stop returning projects at this number
     """
     def get(self, request, *args, **kwargs):
         status = int(request.GET.get('status', 1))
         keywords = request.GET.get('keywords', '').split(',')
         order = request.GET.get('order', 'timestamp')
         salary = request.GET.get('salary', 'lump')
-        ascending = bool(request.GET.get('ascending', False))
+        ascending = bool(int(request.GET.get('ascending', False)))
```

Comment:
You should have some try:catch ValueError: for these because you are dealing with user input.